add content for clinic us server (#9566)
pingcap/docs: clinic/clinic-data-instruction-for-tiup.md

The diagnostic data collected by PingCAP Clinic is **only** used for troubleshooting cluster problems.

> Clinic Server is a diagnostic service deployed in the cloud. There are two independent services bases on different storage location:

Reviewer: ```suggestion
A diagnostic service deployed in the cloud, Clinic Server provides two independent services depending on the data storage location:
```
